.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,32 +1,45 @@
-**Please delete this information after reading it.*
```

Comment:
It was already like this, without the line you are deleting now. The result: even more not modified templates :-| ...

Reply:
The new... Hopefully more obvious line can be seen later in the diff. I'm not sure why Git does this but that's what happened.